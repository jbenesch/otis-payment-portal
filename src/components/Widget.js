import styled from 'styled-components'
import { Colors } from '@blueprintjs/core'

const Widget = styled.div`
  border-radius: 24px;
  background-color: ${Colors.DARK_GRAY3};
  opacity: 0.98;
  width: 100%;
  margin-bottom: 50px;
  padding: 10px;
  padding-bottom: 20px;
`

export default Widget
