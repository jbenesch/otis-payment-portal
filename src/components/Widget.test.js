import React from 'react'
import { render } from 'enzyme'
import Widget from './Widget'

it('matches a previous snapshot', () => {
  const dom = render(<Widget />)
  expect(dom).toMatchSnapshot()
})
