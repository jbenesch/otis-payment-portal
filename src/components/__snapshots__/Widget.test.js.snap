// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches a previous snapshot 1`] = `
<div
  class="sc-bdVaJa dTtgiD"
>
  
</div>
`;
